### ISSUE TITLE
a grammatical problem of c language

### ISSUE THREAD

**Issue by violence-max** [jlu-xiurui/MIT6.S081-2021-FALL#1]
Hi,i am leaning the course as well.when i want to add a defination of 'LAB_PGTBL' at the beginning of the file "maylayout.h" so that i can make use of the content which is written between "#ifdef" and the "endif",but an error come up when i try to excute my whole proj,which reminds me the macros 'LAB_PGTBL' was redefined.i am very curious about how you handle that.

**Comment by jlu-xiurui** [comment]
Hello,if you are learning the course of 2021 Fall,you should note that the "#ifdef LAB_PGTBL" has been defined in 73 line.So that maybe the reason that your code cause a error.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "jlu-xiurui/MIT6.S081-2021-FALL"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年8月18日(星期四) 晚上8:34
***@***.***&gt;;
***@***.***&gt;;
主题:&nbsp;[jlu-xiurui/MIT6.S081-2021-FALL] a grammatical problem of c language (Issue #1)





 
Hi,i am leaning the course as well.when i want to add a defination of 'LAB_PGTBL' at the beginning of the file "maylayout.h" so that i can make use of the content which is written between "#ifdef" and the "endif",but an error come up when i try to excute my whole proj,which reminds me the macros 'LAB_PGTBL' was redefined.i am very curious about how you handle that.

**Comment by violence-max** [comment]
yes I know "#ifdef LAB_PGTBL" has already been difined in line 73 in the file maylayout.h,but,what I want is to use the macros 'USYSCALL ' in other files.The redclareration error occurs when I try to use it as an argument of the function call 'mappages',and I do not kown what to do to avoid the situation.
